lib/application.js

```diff
@@ -26,6 +26,9 @@ var EventEmitter = require('events').EventEmitter,
     Mailer = require('./mailer').Mailer,
     configuration = require('./configuration');
 
+var request = require('request');
+var async = require('async');
+var chalk = require('chalk');
```

Comment:
The install.js uses 'colors' still. Chalk looks better. Can you update the install.js to swap out colors usage for chalk, and remove colors from the dependencies too?

pretty please :)